circle.yml

```diff
@@ -13,7 +13,12 @@ test:
 #   - npm run lint
     - npm test
     - npm run test-mocha
-    - npm run integration
+    - ./node_modules/.bin/integration-loader
```

Comment:
merged loader with setup, since they both come from the `five-bells-integration-test-loader` repo, but left the 4 suites as 4 steps because (as discussed on Slack), a 'non-ilp kit' step with 3 unrelated test suites would still be unhelpful if one of those three suites fail.